etc/profile-m-z/tidal-hifi.profile

```diff
@@ -0,0 +1,38 @@
+# Firejail profile for tidal-hifi
+# Description: The web version of Tidal running in electron with hifi support thanks to widevine.
+# This file is overwritten after every install/update
+# Persistent local customizations
+include tidal-hifi.local
+# Persistent global definitions
+include globals.local
+
+ignore noexec ${HOME}
+
+noblacklist ${HOME}.config/tidal-hifi
+
+include disable-proc.inc
+include disable-shell.inc
+include disable-write-mnt.inc
+
+whitelist ${HOME}.config/tidal-hifi
+
+caps.drop all
+no3d
+nonewprivs
+noprinters
+noroot
+protocol unix,inet,inet6
+seccomp !chroot
+seccomp.block-secondary
```

Comment:
Do you or anyone else know if tidal-hifi always run with no-sandbox. Or do you have unprivileged userns clone enabled (default)?

Reply:
I do have it enabled currently.